quickstart-templates/security-group-add-user-members/README.md

```diff
@@ -0,0 +1,47 @@
+# Configure a security group's user members, referencing users by UPN
+
+This sample demonstrates use of the read-only [`Microsoft.Graph/Users` bicep type][users-ref] which allows you to
+fetch `user` resources by their user principal name (UPN).
+
+This quickstart creates a security group and adds users, referenced via their UPNs, as members.
+The list of users to be added as members are in a txt file, with each user's UPN on a separate line.
+Replace the UPN values in the example "userlist.txt" file with user UPN values from your tenant, before deployment.
+
+## Details
+
+This template sample:
+
+1. Creates a user UPN list from a txt file.
+2. Creates/updates a security group with its members set based on the user UPN list
+
+**NOTE:** Due to current modelling limitations [no more than 20 members can be added/updated at a time][20-members], and only [update semantics][update-only] are supported for members (and owners).
```

Comment:
I don't think it matters much, but this is interesting.  Modelling is the british spelling vs modeling the english spelling.  Is there a doc standard or does it not matter?  The meaning and readability is the same.

Reply:
I have the British English - code spell-checker and the standard extension in VS Code.  I guess I'm running both US and UK...